src/leases/helpers.ts

```diff
@@ -2948,4 +2951,15 @@ export const sortRelatedHistoryItems = (a: Record<string, any>, b: Record<string
   } else {
     return 0;
   }
-};
\ No newline at end of file
+};
+
+/**
+ * The feature for override_receivable_type is only for AKV and KuVa leases.
+ * MaKe users don't want the field visible, to avoid clutter and confusion.
+ *
+ * @param leaseServiceUnitId ID of the lease's service unit
+ * @returns boolean Whether the field should be visible
+ */
+export const showOverrideReceivableTypeField = (leaseServiceUnitId: number): boolean => {
```

Comment:
If serviceunit has a type, the type of the argument here could be ServiceUnit["id"], which would end up with the same type number, but maybe providing more context

Reply:
Ah, didn't know that's possible

Reply:
Not relevant anymore because the function was removed